tests/libmigration/migration.go

```diff
@@ -503,3 +503,17 @@ func ConfirmMigrationMode(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMach
 	By("Verifying the VMI's migration mode")
 	ExpectWithOffset(1, vmi.Status.MigrationState.Mode).To(Equal(expectedMode), fmt.Sprintf("expected migration state: %v got :%v \n", vmi.Status.MigrationState.Mode, expectedMode))
 }
+
+func WaitUntilMigrationMode(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, expectedMode v1.MigrationMode, timeout int, defaultValue v1.MigrationMode) *v1.VirtualMachineInstance {
```

Comment:
I don't think we need to specify the `defaultValue`, it should not be something that the test should manipulate

Reply:
I don't mind, although using the expected "current" mode makes sense. 
We can add it later if that would be needed,